captum/optim/_core/output_hook.py

```diff
@@ -53,8 +26,10 @@ def _reset_outputs(self) -> None:
     def is_ready(self) -> bool:
         return all(value is not None for value in self.outputs.values())
 
-    def _forward_hook(self):
-        def forward_hook(module, input, output) -> None:
+    def _forward_hook(self) -> Callable:
+        def forward_hook(
+            module: nn.Module, input: torch.Tensor, output: torch.Tensor
```

Comment:
JIT is saying that input should have a type hint of `Tuple[torch.Tensor]` instead of just `torch.Tensor`.